<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion commerciale</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabCatalogue">
       <attribute name="title">
        <string>Catalogue</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Catalogue des pieces vehicule</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableViewCatalogue">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>100</number>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonAjoutPiece">
            <property name="text">
             <string>Ajouter</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSupprPiece">
            <property name="text">
             <string>Supprimer</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCommande">
       <attribute name="title">
        <string>Vehicules</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBoxVehicule">
          <property name="title">
           <string>Vehicule</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="labelGestionDesCommandes">
               <property name="text">
                <string>Gestion des vehicules</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="labelImmatriculation">
               <property name="text">
                <string>Immatriculation : </string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEditImmatriculation"/>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="labelMarque">
               <property name="text">
                <string>Marque : </string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="lineEditMarque"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelType">
               <property name="text">
                <string>Type : </string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEditType"/>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="labelCouleur">
               <property name="text">
                <string>Couleur :</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="lineEditCouleur"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="labelCarburant">
               <property name="text">
                <string>Carburant : </string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEditCarburant"/>
             </item>
             <item row="2" column="3">
              <widget class="QPushButton" name="pushButtonAjoutVehicule">
               <property name="text">
                <string>Ajout d'un nouveau vehicule</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="tableViewVehicule">
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>105</number>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>105</number>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxCommande">
          <property name="title">
           <string>Produit</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="labelPiece">
             <property name="text">
              <string>Piece :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxDesignationPiece"/>
           </item>
           <item>
            <widget class="QLabel" name="labelReparation">
             <property name="text">
              <string>Heure main oeuvre : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxHMO"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelQuantite">
             <property name="text">
              <string>Quantité :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxQuantite"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAjoutPiece_2">
             <property name="text">
              <string>&amp;Ajouter</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxPanier">
          <property name="title">
           <string>Visulalisation</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QTableWidget" name="tableWidgetPanier">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>100</number>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>100</number>
             </attribute>
             <column>
              <property name="text">
               <string>Code Piece</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Main d'oeuvre</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Prix unitaire</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Quantité</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>48</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="labelMontant">
               <property name="text">
                <string>Montant Total :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelMontantTotal">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="action_Quitter"/>
   </widget>
   <widget class="QMenu" name="menu_Ev_nements">
    <property name="title">
     <string>&amp;Evènements</string>
    </property>
    <addaction name="on_actionActualiser_le_Catalogue"/>
   </widget>
   <widget class="QMenu" name="menu_Aide">
    <property name="title">
     <string>&amp;Aide</string>
    </property>
    <addaction name="action_propos"/>
   </widget>
   <addaction name="menu_Fichier"/>
   <addaction name="menu_Ev_nements"/>
   <addaction name="menu_Aide"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Quitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
  </action>
  <action name="action_propos">
   <property name="text">
    <string>À &amp;propos</string>
   </property>
  </action>
  <action name="actionActualiser_le_Catalogue">
   <property name="text">
    <string>Actualiser le C&amp;atalogue</string>
   </property>
  </action>
  <action name="actionActualiser_les_C_lients">
   <property name="text">
    <string>Actualiser les C&amp;lients</string>
   </property>
  </action>
  <action name="on_actionActualiser_le_Catalogue">
   <property name="text">
    <string>&amp;Rafrechir le catalogue</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
